api/src/main/java/org/pmiops/workbench/google/CloudStorageServiceImpl.java

```diff
@@ -32,6 +34,16 @@ public String readBlockscoreApiKey() {
     return readToString(getCredentialsBucketName(), "blockscore-api-key.txt").trim();
   }
 
+  public String readMandrillApiKey() {
+    JSONObject mandrillKeys = new JSONObject(readToString(getCredentialsBucketName(), "mandrill-keys.json").trim());
```

Comment:
One thing I'd like to see us do is also try to refactor when possible. For example, look at all of these `readToString` calls and note that they all use `.trim()` at the end. Instead of following the same duplicative pattern, we should try to clean these up. In this case, if we updated that method to be
```
  String readToString(String bucketName, String objectPath) {
    Storage storage = StorageOptions.getDefaultInstance().getService();
    return new String(storage.get(bucketName, objectPath).getContent()).trim();
  }
```
we could remove all the extra `trim()`s in each of the calls to `readToString`. It's a small change, but I'd like us to get into the habit of incremental and continual improvement.